our_repo.py

```diff
@@ -80,24 +83,24 @@ def number_of_016_files(self):
 
         return elm_016_count
 
-    def get_files_for_017(self, pattern):
+    def get_files_for_017(self, pattern: str) -> List[str]:
         pattern = pattern.replace('.', '/')
         all_files = glob.glob(f'{self.repo_dir}/**/{pattern}.elm', recursive=True)
 
-        return [ file for file in all_files
-            if self.what_kinda_file(file) == ELM_017_FILE
+        return [ filename for filename in all_files
+            if self.what_kinda_file(filename) == ELM_017_FILE
         ]
 
-    def get_files_for_017_with_breakdown(self, pattern):
+    def get_files_for_017_with_breakdown(self, pattern: str) -> Dict[str,Dict[str, int]]:
```

Comment:
💭 should be `get_files_for_016_with_breakdown`? or

- `get_016_files_with_breakdown`
- `get_017_porting_breakdown`